src/w3c/templates/sotd.js

```diff
@@ -121,8 +121,15 @@ function renderIsNoTrack(conf, opts) {
 
 function renderNotRec(conf) {
   let statusExplanation = "";
-  let updatePolicy =
-    "This document may be updated, replaced or obsoleted at any time. It is inappropriate to cite this document as other than work in progress. ";
+  let updatePolicy = html`This is a draft document and may be updated, replaced
+  or obsoleted by other documents at any time. It is inappropriate to cite this
+  document as other than work in progress.
+  ${conf.updateableRec
+    ? html`Future updates to this specification may incorporate
+        <a href="https://www.w3.org/2020/Process-20200915/#revised-rec-features"
+          >new features</a
+        >.`
+    : ""}`;
```

Comment:
Nit: Maybe we can assign this to a variable and remove the duplication below?

Reply:
I'll take that as a followup.